destination-db/database-objects/010_createTables.sql

```diff
@@ -294,4 +294,11 @@ CREATE TABLE IF NOT EXISTS TEST_CATEGORY
 	TEST_CATEGORY_REF VARCHAR(10) NOT NULL,
 	THEORY_TYPE_CODE INT NOT NULL,
 	CONSTRAINT PK_TEST_CATEGORY PRIMARY KEY (TEST_CATEGORY_REF)
+);
+
+CREATE TABLE IF NOT EXISTS DES_TEST_CRITERIA (
```

Comment:
EXAMINER_STAFF_NUMBER and WITH_EFFECT_TO are nullable

Reply:
Marked the inverse as `NOT NULL`.